python/dgl/graphbolt/negative_sampler.py

```diff
@@ -120,4 +165,22 @@ def _collate(self, minibatch, neg_pairs, etype=None):
                 minibatch.negative_srcs = neg_src
                 minibatch.negative_dsts = neg_dst
         else:
```

Comment:
For in-place function update, feel free to update the doc string according to the new implementation.